src/viur/core/skeleton.py

```diff
@@ -1330,9 +1387,11 @@ def txnDelete(skel: SkeletonInstance):
         for boneName, _bone in skel.items():
             _bone.postDeletedHandler(skel, boneName, key)
         skel.postDeletedHandler(key)
+
         # Inform the custom DB Adapter
-        if skel.customDatabaseAdapter:
-            skel.customDatabaseAdapter.deleteEntry(dbObj, skel)
+        for adapter in skel.customDatabaseAdapter:
+            adapter.deleteEntry(dbObj, skel)  # FIXME: DEPRECATED: Superseeded by adapter.trigger.
```

Comment:
We decided in viur-meeting to keep both.
```suggestion
            adapter.deleteEntry(dbObj, skel)
```